.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## Reason for change
+
+<!-- Describe here the reason for change, and provide a link to the corresponding ClickUp task or Sentry issue. -->
+
+[ClickUp task](https://app.clickup.com/20427503/v/b/4-32476982-2/CLICKUP_ISSUE_ID)
```

Comment:
https://app.clickup.com/20427503/v/b/4-32476982-2 is the global integration backlog URL
maybe you should use https://app.clickup.com/20427503/v/b/6-140492922-2 for the woocommerc backlog 😘